Source/OxyPlot/PlotController/Manipulators/TrackerManipulator.cs

```diff
@@ -47,6 +49,16 @@ public TrackerManipulator(IPlotView plotView)
         /// <value><c>true</c> if the tracker should be locked; otherwise, <c>false</c>.</value>
         public bool LockToInitialSeries { get; set; }
 
+        /// <summary>
+        /// Gets or sets the distance from the series at which the tracker fires.
+        /// </summary>
+        public double FiresDistance { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to check distance when showing tracker between data points.
+        /// </summary>
+        public bool IsCheckDistanceBetweenPoints { get; set; }
```

Comment:
Again, this name is a bit weird (and it's public). I know you don't like `Should`, but it sounds wrong to me with `Is`. Can I suggest `CheckDistanceBetweenPoints`? There hasn't been any strict adherence to prefixing boolean properties previously.